examples/bri-3/src/bri/transactions/agents/transactions.agent.spec.ts

```diff
@@ -1,330 +1,339 @@
-import { DeepMockProxy, mockDeep } from 'jest-mock-extended';
-import { uuid } from 'uuidv4';
-import { ICircuitService } from '../../../bri/zeroKnowledgeProof/services/circuit/circuitService.interface';
-import { TestDataHelper } from '../../../shared/testing/testData.helper';
 import { AuthAgent } from '../../auth/agent/auth.agent';
-import { BpiSubjectAccount } from '../../identity/bpiSubjectAccounts/models/bpiSubjectAccount';
-import { BpiSubject } from '../../identity/bpiSubjects/models/bpiSubject';
+import { BpiSubjectAccount as ModelBpiSubjectAccount } from '../../identity/bpiSubjectAccounts/models/bpiSubjectAccount';
```

Comment:
it is the other way around in the prisma mapper when it comes to naming.
Domain objects imported without aliases.
PrismaModels have PrismaModel suffix.
Could we do the same here and use this as a convention throughout the codebase?

We can also turn it around, i do not care as long as we are consistent